src/main/java/org/opensearch/security/privileges/SystemIndexAccessEvaluator.java

```diff
@@ -56,7 +59,7 @@
  * - The term `protected system indices` used here translates to system indices
  *   which have an added layer of security and cannot be accessed by anyone except Super Admin
  */
-public class SecurityIndexAccessEvaluator {
+public class SystemIndexAccessEvaluator {
```

Comment:
Should we officially deprecated [ProtectedIndicesAccessEvaluator](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/privileges/ProtectedIndexAccessEvaluator.java)?

Reply:
I think we still need [ProtectedIndicesAccessEvaluator](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/privileges/ProtectedIndexAccessEvaluator.java) for non-system indices.

Reply:
okay, I have not seen that being actively used but maybe I'm not aware of the entire flow.